solr/solr-ref-guide/src/field-type-definitions-and-properties.adoc

```diff
@@ -87,6 +87,13 @@ For multivalued fields, specifies a distance between multiple values, which prev
 
 `autoGeneratePhraseQueries`:: For text fields. If `true`, Solr automatically generates phrase queries for adjacent terms. If `false`, terms must be enclosed in double-quotes to be treated as phrases.
 
+`synonymQueryStyle`:: 
+Query used to combine scores of overlapping query terms (ie synonyms). Consider a search for "blue tee" with query-time synonyms `tshirt,tee`.
++
+Use `as_same_term` (default) to blend terms, ie `SynonymQuery(tshirt,tee)` where each term will be treated as equally important. Use `pick_best` to select the most significant synonym when scoring `Dismax(tee,tshirt)`. Use `as_distinct_terms` to bias scoring towards the most significant synonym `(pants OR slacks)`.
++
+`as_same_term` is appropriatte when terms are true synonyms (television, tv). `pick_best` and `as_distinct_terms` are appropriatte when synonyms are expanding to hyponyms (q=jeans w/ jeans=>jeans,pants) and you want exact to come before parent and sibling concepts. See this http://opensourceconnections.com/blog/2017/11/21/solr-synonyms-mea-culpa/[blog article].
```

Comment:
Is "appropriate" spelled wrong (with an extra 't')? It's done twice so I'm not sure if I'm perhaps misunderstanding the context.

Reply:
Thanks @ctargett, this is one of those words I consistently misspell. Github spellchecking failed me, so I brought it down and double checked/fixed the spelling.